consensus/polybft/consensus_runtime.go

```diff
@@ -743,20 +755,18 @@ func (c *consensusRuntime) InsertBlock(proposal []byte, committedSeals []*messag
 		return
 	}
 
-	if c.IsBridgeEnabled() {
-		if (fsm.isEndOfEpoch || c.checkpointManager.isCheckpointBlock(fullBlock.Block.Header.Number)) &&
-			bytes.Equal(c.config.Key.Address().Bytes(), fullBlock.Block.Header.Miner) {
-			go func(header types.Header, epochNumber uint64) {
-				if err := c.checkpointManager.submitCheckpoint(header, fsm.isEndOfEpoch); err != nil {
-					c.logger.Warn("failed to submit checkpoint",
-						"checkpoint block", header.Number,
-						"epoch number", epochNumber,
-						"error", err)
-				}
-			}(*fullBlock.Block.Header, fsm.epochNumber)
-
-			c.checkpointManager.latestCheckpointID = fullBlock.Block.Number()
-		}
+	if c.checkpointManager.IsCheckpointBlock(fullBlock.Block.Header.Number, fsm.isEndOfEpoch) &&
```

Comment:
This can be done on a follow-up PR but I assume we are also going to move this also on a PostBlock op right? That will simplify the `checkpointManager` function to only have the handles as far as I can see.

Reply:
Indeed. Didn't want to do all the work through this PR, since we have 2 more tasks for checkpoints for RFC-176 prep.

Reply:
Cool. That is what I thought too.

Reply:
Ok I was wondering the same: why this part isn't moved to `checkpointManager`. When it is moved away to `PostBlock`, then we can remove `IsCheckpointBlock` function from `CheckpointManager` interface.